src/components/ADempiere/Form/ProductSearch/dialogo/infoProduct.vue

```diff
@@ -126,6 +126,7 @@ export default defineComponent({
      */
 
     const product = computed(() => {
+      console.log({ product: store.getters.getCurrentProduct })
```

Comment:
please remove this `console.log`

Reply:
https://github.com/solop-develop/frontend-core/pull/2770/commits/9dec40da52f4127d13efafc64c0ff4e3901624dc